cmake/CMakeLists.txt

```diff
@@ -218,6 +219,10 @@ if (protobuf_BUILD_TESTS)
   include(tests.cmake)
 endif (protobuf_BUILD_TESTS)
 
+if (protobuf_BUILD_CONFORMANCE)
+  include(conformance.cmake)
```

Comment:
This new file needs to be included in the toplevel Makefile.am file for the make dist test to pass.

Reply:
Done.